core/templates/dev/head/domain/skill/SkillObjectFactory.ts

```diff
@@ -55,6 +59,11 @@ angular.module('oppia').factory('SkillObjectFactory', [
         issues.push(
           'There should be review material in the concept card.');
       }
+      if (this.getRubrics().length !== 3) {
```

Comment:
Maybe verify the specific keys too, and the ordering.

Your frontend form can force the ordering, btw (e.g. have three fields pre-labelled with easy/medium/hard) so that this trigger never occurs due to user error (i.e., try to eliminate the possibility of user error)

Reply:
I've done the second way, by forcing the user to only fill in for predefined difficulties.